tests/pact/ops.repl

```diff
@@ -147,7 +147,7 @@
 (expect "!= moduleGuard moduleGuard" true (!= (tm.mk "2") (tm.mk "1")))
 
 "===== pact guard equality"
-(env-hash "aa")
+(env-hash "YQo")
```

Comment:
A small fix along the way

Reply:
Why does the transaction hash change?

Reply:
When using a newer, less lenient base64 version, I got an error from base64 complaining that the input "aa" isn't valid base64. I assumed that "aa" is just a mock for some base64 encoded hash. But not 100% sure.

Reply:
Where is the base64 change?

Reply:
It showed up in a build with base64-bytestring-1.2.0.0. I attributed it to this entry from the [CHANGELOG](https://hackage.haskell.org/package/base64-bytestring-1.2.0.0/changelog):

```
Security fix: reject non-canonical base64 encoded values - (#38) fixing issue #24.
```

Reply:
The stack and nix builds uses an older version of base64-bytestring, so the issue doesn't show up in those builds.

Reply:
I added a separate test case and a temporary upper bound on base64-bytestring.